src/BannerViewManager.js

```diff
@@ -32,6 +32,7 @@ const getSizeForType = (type: AdType) => sizeForType[type] || sizeForType.standa
 type BannerViewProps = {
   type: AdType,
   placementId: string,
+  style: { [key: string]: any },
```

Comment:
@jakubsta `Stylesheet.create(...)` is returning `StyleSheet<S>` where `S: Styles` which is `{ [key: string]: Object }`. But if we use what was in flow-typed issue we can defined this as `StyleSheet.Styles` and everything passes correctly.